tests/unit/combine_results/combined_MotEvo_results.csv

```diff
@@ -0,0 +1,22 @@
+strand,binding_position,binding_posterior,pwm_id,binding_region,binding_sequence,binding_energy,fasta_record
++,14-18,0.015291,HNRNPF_820,"MOTEVO_input-sequence
+",GTGGG,2.51049,"MOTEVO_input-sequence
```

Comment:
I am very confused by the information here... why is every record separated into two rows? Each record should be one line (i.e. one row) in the result table. The records should match the header of the table.